campaigns/001-netperf-data-exchange/run.py

```diff
@@ -35,6 +40,11 @@ def main(x):
     # usage: host id:[string] mode:[start|stop] local-file-name:"path_and_filename" filter:"tcpdump filter string"
     x.exec('006-tcpdump beta id:0001 mode:start local-file-name:"ignored" filter:"tcp and dst port 30000"')
 
+    # netem
+    # add delay to left and right outgoing interfaces for the upcoming test
+    x.exec('104-netem-cmd koppa control:part change:add device:left command:"delay 10ms"')
```

Comment:
left? towards-alpha, to-alpha -> better! Now I know, left, right? I don't know what it means? Is right towards alpha, or beta? Change!